server/logger/logger.go

```diff
@@ -0,0 +1,62 @@
+package logger
+
+import (
+	"log"
+	"os"
+	"io/ioutil"
+	"io"
+)
+
+const loggerFlags = log.Ldate | log.Ltime
+
+
+type Logger struct{
+	debugLogger log.Logger
+	infoLogger log.Logger
+	warningLogger log.Logger
+	errorLogger log.Logger
+	fatalLogger log.Logger
+}
+
+
+func InitializeLogger(level uint8) *Logger {
```

Comment:
Możemy to nazwać po prostu "New(...)", z nazwy paczki można się domyślić o co chodzi